frontend/src/metabase/visualizations/components/FunnelNormal.jsx

```diff
@@ -65,6 +65,9 @@ export default class FunnelNormal extends Component {
 
     let remaining = rows[0][metricIndex];
 
+    console.log(cols, metricIndex);
+    console.log(rows);
```

Comment:
Can we remove the logs?

Reply:
![200](https://user-images.githubusercontent.com/1328979/186910286-34931860-5262-4b35-b52d-e0cf63ad6d8f.gif)